Source/ablastr/math/fft/WrapFFTW.cpp

```diff
@@ -67,9 +74,12 @@ namespace ablastr::math::anyfft
             } else if (dim == 2) {
                 fft_plan.m_plan = VendorCreatePlanC2R2D(
                     real_size[1], real_size[0], complex_array, real_array, FFTW_ESTIMATE);
+            } else if (dim == 1) {
+                fft_plan.m_plan = VendorCreatePlanC2R1D(
+                    real_size[0], complex_array, real_array, FFTW_ESTIMATE);
             } else {
                 ABLASTR_ABORT_WITH_MESSAGE(
-                    "only dim=2 and dim=3 have been implemented. Should be easy to add dim=1.");
+                    "only dim=1 and dim=2 and dim=3 have been implemented.");
```

Comment:
Do we want to keep this general for possible future scenarios with `dim` > 3? If not, maybe we could also remove these last `else` blocks.

Reply:
If we remove the last `else` block, then we will silently do nothing on other dims than 1,2,3 passed by a user (could be by accident). Thus, I would keep it to explicitly handle all cases.